### DISCUSSION TITLE
<title>Graphics Problem and DirectDraw

### DISCUSSION THREAD

**Discussion by TheBoxGame** [community/community#114566] [Pull Requests]
### Select Topic Area

General

### Body

Hello, I have a question whether the DirectDraw option will be added in Winlator so that the games run in full screen even after changing the resolution and whether the problem with some games such as this one will be fixed, I am providing screenshots of what it should look like
![Screenshot_2024-03-28-09-25-50-700_com winlator](https://github.com/community/community/assets/109180532/70d10c83-c3f1-4132-9fc5-fa1ce67d8862)
![Screenshot_2024-03-28-09-28-51-299_com google android youtube](https://github.com/community/community/assets/109180532/4ea5757b-64c4-4f1a-8ce6-eab43b55c877)

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by TheBoxGame** [comment]
@queenofcorgis and could you write to the support to add the DirectDraw option and fix the graphics problem because I don't know how to write to the support

**Reply by queenofcorgis** [reply]
@TheBoxGame The GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. 

GitHub is a software hosting service, we host millions of projects. We're not able to provide support or answers for those individual projects. GitHub Support and Community Discussions are only able to provide support for GitHub itself.

**Reply by TheBoxGame** [reply]
ok

**Comment by TheBoxGame** [comment]
@queenofcorgis I wanted to fix it in the next update
